### ISSUE TITLE
Change "p" and "q" get parameters

### ISSUE THREAD

**Issue by kaedroho** [torchbox/wagtail#1064]
The admin index views all use "p" for page number and "q" for the search query. I think these should be changed to make them more clearer.

Suggestion:
 - Page number: "page"
 - Search query: "search"

**Comment by kaedroho** [comment]
#1315

**Comment by lb-** [comment]
@kaedroho  - long time since this was created, do you still think this is something worth looking at?

And are you thinking the same for modeladmin also?

**Comment by lb-** [comment]
Also - what about ordering? `order` (seems most common)

See related #5191

**Comment by allcaps** [comment]
The `q` query string param seems a convention for search. I would not change it as it isn't unclear. Maybe it is just not unclear to me?

`page` and `order` are not so obvious. Readability counts.

Do we need to worry about backward compatibility?

**Comment by lb-** [comment]
I agree, `q` seems pretty universal, and backwards compat for a time might help, even if it has a warning somehow.

This might become quite a big task but I agree it will make URLs more readable which I think helps alot.
